autogpt_platform/frontend/src/components/integrations/credentials-input.tsx

```diff
@@ -86,6 +87,8 @@ export const providerIcons: Record<
   exa: fallbackIcon,
   hubspot: FaHubspot,
   todoist: fallbackIcon,
+  smartlead: fallbackIcon,
```

Comment:
nit, but I see others are already out-of alphabetical order ;(
```suggestion
  smartlead: fallbackIcon,
  todoist: fallbackIcon,
```